docs/user-guides/python_api.md

```diff
@@ -246,12 +246,17 @@ setpoint with the same thrust but with roll/pitch/yawrate = 0, this
 will make the Crazyflie stop accelerating. After 2 seconds without new
 setpoint the Crazyflie will cut power from the motors.
 
+
 Note that this command implements a motor lock mechanism that is
 intended to avoid flyaway when connecting a gamepad. You must send
 one command with thrust = 0 in order to unlock the command. This
 unlock procedure needs to be repeated if the watchdog described above
 kicks-in.
 
+This the current default behavior, but it is depended on the [Supervisor configuration in the Crazyflie firmware](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/supervisor/).
```

Comment:
This _is_ the current default behavior, but it is _dependent_ / but it _depends_ on

Reply:
thanks! You can use suggestions by the way in the reviews. I'm using it now to quickly implement these edits

```suggestion
This the current default behavior, but it depends on the [Supervisor configuration in the Crazyflie firmware](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/supervisor/).
```